test-omni-integ-regtest.sh

```diff
@@ -9,7 +9,7 @@ trap cleanup EXIT
 BTCD=copied-artifacts/src/omnicored
 DATADIR=$HOME/.bitcoin
 LOGDIR=logs
-MSCLOG=/tmp/mastercore.log
+MSCLOG=/tmp/omnicore.log
```

Comment:
@msgilligan: does this interfer with the Jenkins setup (i.e. does Jenkins expect `mastercore.log`)?